spec/factories.rb

```diff
@@ -16,6 +16,7 @@
   end
 
   factory :delivery_attempt do
+    id { SecureRandom.uuid }
```

Comment:
I'm not sure you need this - unless we're relying on this existing for some `build` methods?